src/intl/messages.js

```diff
@@ -699,8 +726,10 @@ export const messages: { [messageId: string]: MessageType } = {
   utilizationNoDataAvailableMessage: 'Utilization data is only available when the VM is running.',
   utilizationNoHistoricData: 'No historic data available',
   utilizationNoNetStats: 'Network utilization is not currently available for this VM.',
+  usbAutoshare: 'USB Auto-Share',
```

Comment:
This is not required/planned as part of console options

Reply:
Added just in case to be compliant with Web Admin console options.

Reply:
Same here